### PR TITLE
Major improoooovments

### PR DESCRIPTION
**Pull request by maethu**
Major changes
- Upload panes using quickupload
- Edit panes directly in the slider block

Watch this:
slider

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,6 +5,9 @@ Changelog
 1.0.3 (unreleased)
 ------------------
 
+- This release needs at lease ftw.simplelayout 1.12.0.
```
Comment (jone) [inline docs/HISTORY.txt:8]: 🚧 This should be declared in the dependencies (`ftw.simplelayout [contenttypes] >= 1.12.0`) as soon as `ftw.simplelayout` is released.

File: ftw/sliderblock/browser/resources/styles.scss [hunk 2]
```diff
@@ -1 +1,11 @@
 @include portal-type-font-awesome-icon(ftw-sliderblock-sliderblock, forward);
+
+.ftw-sliderblock-sliderblock .inneredit {
+  font-size: $font-size-large;
+  line-height: $font-size-large;
+  position: absolute;
+  right: 0;
+
```
Comment (jone) [inline ftw/sliderblock/browser/resources/styles.scss:8]: The icon is often barely visible, depending on the image:
bildschirmfoto 2016-10-20 um 12 15 57

I propose to use a ligther text color and apply a dark outer glow effect for having more contrast, making the icon better visible on most images, without having to apply a background color or moving the icon out of the image.

``` css
  &:before {
    color: #EEE;
    text-shadow: 1px 0px 1px rgba(136, 136, 136, 0.7), 0px 1px 1px rgba(136, 136, 136, 0.7), -1px 0px 1px rgba(136, 136, 136, 0.7), 0px -1px 1px rgba(136, 136, 136, 0.7);
  }
  &:hover:before {
    color: #FFF;
    text-shadow: 1px 0px 1px rgba(136, 136, 136, 1), 0px 1px 1px rgba(136, 136, 136, 1), -1px 0px 1px rgba(136, 136, 136, 1), 0px -1px 1px rgba(136, 136, 136, 1);
  }
```

bildschirmfoto 2016-10-20 um 12 16 28

**Review by jone — CHANGES_REQUESTED** [review]

File: docs/HISTORY.txt (hunk 1 above)
Comment (maethu) [reply docs/HISTORY.txt:8]: Yep I'll do that, I'm no yet sure if it is 1.12.0, or something else 😉

File: ftw/sliderblock/browser/resources/styles.scss (hunk 2 above)
Comment (maethu) [reply ftw/sliderblock/browser/resources/styles.scss:8]: @jone Probably a white slightly opac background, with a dark icon would be the best. Now it's also hard to see on the image with house/tree

File: docs/HISTORY.txt (hunk 1 above)
Comment (maethu) [reply docs/HISTORY.txt:8]: @jone add the requirement 1.12.0 it is :-)

**Review by jone — APPROVED** [review]

**Pull request merged**